src/machine/behavior/is/float/fint.rs

```diff
@@ -1,6 +1,22 @@
 use machine::state::State;
 
-pub fn fint(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn fint(state: &mut State, x: u8, y: u8, z: u8) {
+    // Load operands
+    let op1: f64 = state.gpr[z].into();
+
+    // Execute
+    let mut res: f64;
+    match y {
+        1 => res = op1.round(),    // round off
+        2 => res = op1.ceil(),     // round up
+        3 => res = op1.floor(),    // round down
+        4 => {                     // round near to ten
+            res = (op1 + 5.0) / 10.0;
+            res = res.floor() * 10.0;
+        },
+        _ => res = op1.round(),    // round off
```

Comment:
Ok I know what you mean. Sounds as if it's better to panic. As I read the declaration I thought it's also possible to call the instruction without a rounding mode but it makes no sense. I will change the default case.